diesel/src/sqlite/connection/mod.rs

```diff
@@ -132,6 +132,16 @@ impl SqliteConnection {
         }
         query.map(|_| ())
     }
+
+    #[doc(hidden)]
+    pub fn execute_pragma<ST, U>(&self, source: &str) -> QueryResult<Vec<U>>
+        where U: Queryable<ST, Sqlite>,
+              <SqliteConnection as Connection>::Backend: HasSqlType<ST>
+    {
+        Statement::prepare(&self.raw_connection, source)
+            .map(StatementIterator::new)
+            .and_then(Iterator::collect)
+    }
```

Comment:
To move this out of the Diesel main crate I will need to expose raw_connection. This seems like a bad idea for me.
The alternative implementation of this would be to add a pragma function, that act like select but add `Pragma` instead (without any from part).